libnavigation-core/src/main/java/com/mapbox/navigation/core/internal/dump/MapboxDumpCommand.kt

```diff
@@ -0,0 +1,18 @@
+package com.mapbox.navigation.core.internal.dump
+
+data class MapboxDumpCommand(
+    val command: String,
+    val values: List<MapboxDumpCommand>? = null
```

Comment:
It's not going to stay that way. This is in a work in progress to be honest, the input is translated in a slightly confusing way. I also recently discovered it will not accept json arrays, but accepts some json. Here is an example where json is kinda translated.

```
➜  mapbox-navigation-android git:(km-update-unit-type-from-command-line) ✗  adb shell dumpsys activity service com.mapbox.navigation.core.trip.service.NavigationNotificationService "animal":{"age":4,"name":"Felis catus","weight":{"units":"kilograms","value":4.5}}
```
```
  "animal": {
    "age": 4,
    "name": "Felis catus",
    "weight": {
      "units": "kilograms",
      "value": 4.5
    }
  }
```

```
    args[0] = animal:age:4
    args[1] = animal:name:Felis
    args[2] = catus
    args[3] = animal:weight:units:kilograms
    args[4] = animal:weight:value:4.5
```

This is why I moved it to draft and will add test cases

Reply:
Ok added examples inside the code and descriptions.